org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateAdaptorManager.mtl

```diff
@@ -120,6 +144,19 @@ public class [javaClassNameForAdaptorManager(anAdaptorInterface) /] {
         // [protected ('contextInitializeServletListener')]
         // TODO Implement code to establish connection to data backbone etc ...
         // [/protected]
+        [if (anAdaptorInterface.trsServiceEnabled())]
+        // [protected ('TRSInitialise')]
+        // [/protected]
+        ArrayList<URI> uris = new ArrayList<URI>();
+        // [protected ('TRSInitialBase')]
+        //TODO: Provide the initial list of URIs to populate the TRS log with
+        // [/protected]
+        InmemPagedTrs temp = new PagedTrsFactory().getInmemPagedTrs(50, 50, uris);
```

Comment:
are you going to use `basePageLimit` property defined in the model?

Reply:
You’re good. You can debug by looking at this code? 
I’ll fix. I tested using the same value (50) so never detected the problem :-)